### PR TITLE
Update to latest LTS and other fixes

### PR DESCRIPTION
**Pull request by pbrisbin**
Please see individual commits.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ release: build/popeye
 	docker build --tag $(IMAGE_NAME) .
 
 check: release
-	docker run --rm --env-file .env \
+	docker run --rm --volume ~/.aws:/root/aws:ro \
```
Comment (ale7714) [inline Makefile]: does this mean that if we want to try this locally we would need our credentails on an `aws` directory?

**Review by ale7714 — APPROVED** [review]
LGTM! just a small question... will merge soon

File: Makefile (hunk 1 above)
Comment (pbrisbin) [reply Makefile:17]: Correct, my hope was that expecting `~/.aws` to exist is actually less brittle an expectation than the current one of looking for `~/.env`.

`~/.aws` isn't some special thing (like `.env` was), it'll exist and be right provided you've used the `aws` CLI locally at some point (and so done `aws configure`).

Comment (ale7714) [reply Makefile:17]: yeah, thanks! I try it locally and it's working as expected :)

**Pull request merged**